src/queue/job/v1/send_invitation_email.rs

```diff
@@ -0,0 +1,69 @@
+use crate::{
+    entity::*,
+    queue::{Job, JobError, SharedJobState, V1},
+};
+use sea_orm::{ConnectionTrait, EntityTrait};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use url::Url;
+use uuid::Uuid;
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct SendInvitationEmail {
+    pub membership_id: Uuid,
+    pub action_url: Url,
+    pub message_id: Uuid,
+}
+
+impl SendInvitationEmail {
+    pub async fn perform(
+        &mut self,
+        job_state: &SharedJobState,
+        db: &impl ConnectionTrait,
+    ) -> Result<Option<Job>, JobError> {
+        let (membership, account) = Memberships::find_by_id(self.membership_id)
+            .find_also_related(Accounts)
+            .one(db)
+            .await?
+            .ok_or_else(|| {
+                JobError::MissingRecord(String::from("membership"), self.membership_id.to_string())
+            })?;
+
+        let account = account.ok_or_else(|| {
+            JobError::MissingRecord(String::from("account"), membership.account_id.to_string())
+        })?;
+
+        job_state
+            .postmark_client
+            .send_email_template(
+                &membership.user_email,
+                "user-invitation",
+                &json!({
+                    "email": membership.user_email,
+                    "account_name": &account.name,
+                    "action_url": self.action_url
+                }),
+                Some(format!("{}@divviup.org", self.message_id)),
```

Comment:
nit: might be nice to have a config option governing the domain we send email from. I'm OK with filing an issue to add such an option in the future, though, as this PR is already quite large.

Reply:
addressed in [47ec74d](https://github.com/divviup/divviup-api/pull/67/commits/47ec74db3939a12ac35883c82c18dda40dcbb800)